### PR TITLE
depends: qt 5.15.13

### PR DESCRIPTION
**Pull request by fanquake**
I need to try and patch around Qts buildsystem to keep #21778 moving along (the issue being that even when you tell Qt to build using Clang on Linux, it still calls out to GCC, breaking our ability to have a macOS release build env that doesn't have a GCC toolchain installed, and thus no `ld` binary).

Before trying to patch Qt any further, update to the latest LTS release, and update the current patch set.

### CODE REVIEW

**Comment by hebasto** [comment]
Concept ACK.

File: depends/patches/qt/use_android_ndk23.patch [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-+++ new/qtbase/mkspecs/features/android/default_pre.prf
-@@ -76,7 +76,7 @@ else: equals(QT_ARCH, x86_64): CROSS_COMPILE = $$NDK_LLVM_PATH/bin/x86_64-linux-
- else: equals(QT_ARCH, arm64-v8a): CROSS_COMPILE = $$NDK_LLVM_PATH/bin/aarch64-linux-android-
- else: CROSS_COMPILE = $$NDK_LLVM_PATH/bin/arm-linux-androideabi-
- 
--QMAKE_RANLIB            = $${CROSS_COMPILE}ranlib
-+QMAKE_RANLIB            = $$NDK_LLVM_PATH/bin/llvm-ranlib
- QMAKE_LINK_SHLIB        = $$QMAKE_LINK
- QMAKE_LFLAGS            =
-
```
Comment (hebasto) [inline depends/patches/qt/use_android_ndk23.patch:13]: A note for reviewers:
The related change was backported in https://github.com/qt/qtbase/commit/03262625a2ad1133efd19d1e4a73962fb386dfde.

**Comment by fanquake** [comment]
Guix Build (x86_64 & aarch64):
```bash
cdb3ff2cc9ee2ac4a60c8921359396c7b5b83447b54d223fa767c3e641bac182  guix-build-ac9879c061af/output/aarch64-linux-gnu/SHA256SUMS.part
58befd66969efb6473dc68ad9476334bf7544c4827ec28eae28437263b248020  guix-build-ac9879c061af/output/aarch64-linux-gnu/bitcoin-ac9879c061af-aarch64-linux-gnu-debug.tar.gz
53c6dd645f81be3bc3f52768331503c67034db17618fedd344e1283e50700caa  guix-build-ac9879c061af/output/aarch64-linux-gnu/bitcoin-ac9879c061af-aarch64-linux-gnu.tar.gz
7b9aa3aaa5ec51c7d8036abb94eb1f23195049022ee43ee11d401bc86f2990d9  guix-build-ac9879c061af/output/arm-linux-gnueabihf/SHA256SUMS.part
e67d97e27791d8323a4e958dba07d8b5ad220d20d525b77c223e1d13a139e555  guix-build-ac9879c061af/output/arm-linux-gnueabihf/bitcoin-ac9879c061af-arm-linux-gnueabihf-debug.tar.gz
bf621a9c9bf2f5a2a827e552766df7de6f0f4610f8d22a9b776b47e9f878af04  guix-build-ac9879c061af/output/arm-linux-gnueabihf/bitcoin-ac9879c061af-arm-linux-gnueabihf.tar.gz
fb2ecccd91e4154a47d7f509bd4f8255a3fd99e5bcc0af8a0d94ac7a16dd083f  guix-build-ac9879c061af/output/arm64-apple-darwin/SHA256SUMS.part
a3d592d77e231d8fded035a9f388f7c19285178c301d7c8e75e1510ad4ad399d  guix-build-ac9879c061af/output/arm64-apple-darwin/bitcoin-ac9879c061af-arm64-apple-darwin-unsigned.tar.gz
7d6bf27ee6568e4507847cb9d46dc25e35e79d9b1441807003d41b26a962ec5f  guix-build-ac9879c061af/output/arm64-apple-darwin/bitcoin-ac9879c061af-arm64-apple-darwin-unsigned.zip
84ed55538e7ff2baf9436fb277e9a92a2791e17a976b9ef8316daf36fc65096b  guix-build-ac9879c061af/output/arm64-apple-darwin/bitcoin-ac9879c061af-arm64-apple-darwin.tar.gz
9ba85fd31ebc9b7056345fb33b1ecdfe2c2b5be0fb34900be4f680145dcd705c  guix-build-ac9879c061af/output/dist-archive/bitcoin-ac9879c061af.tar.gz
c4a82c0e42e5f950d1c11df0e4fb17b417bd402261103ee409fad52d94042a00  guix-build-ac9879c061af/output/powerpc64-linux-gnu/SHA256SUMS.part
4bfa02a566b0862a027eca2d2b72746f3202dd99a20f912d05532e2d552a30bb  guix-build-ac9879c061af/output/powerpc64-linux-gnu/bitcoin-ac9879c061af-powerpc64-linux-gnu-debug.tar.gz
4594b1f0b8a64f795c7d1b88071e1844f7580cb8217973673212a533d218eb68  guix-build-ac9879c061af/output/powerpc64-linux-gnu/bitcoin-ac9879c061af-powerpc64-linux-gnu.tar.gz
790d95eb1a55d6bfe540abd713004118158480438a2a80d468a7fd781ff1f8a2  guix-build-ac9879c061af/output/riscv64-linux-gnu/SHA256SUMS.part
d54493b26d33cd144cf0376669efba365e6fc59cc2e4bf8bbdaf374373067460  guix-build-ac9879c061af/output/riscv64-linux-gnu/bitcoin-ac9879c061af-riscv64-linux-gnu-debug.tar.gz
8bd52003a4b01660916a97234ab172d3b8d4bfe90deea9025db54648c977f274  guix-build-ac9879c061af/output/riscv64-linux-gnu/bitcoin-ac9879c061af-riscv64-linux-gnu.tar.gz
38bc5ba2db9a2baf8b4be8cd80571a733ce11396a183313d2e5eda4390e25e12  guix-build-ac9879c061af/output/x86_64-apple-darwin/SHA256SUMS.part
6a71afeef1b80640cb830ab6c3b2fc4b3b5636dd20eb927aac39576e741f4dda  guix-build-ac9879c061af/output/x86_64-apple-darwin/bitcoin-ac9879c061af-x86_64-apple-darwin-unsigned.tar.gz
f9f9cddcfe2ca78998c77aa2feb580a5dfbd7e04b14a52a6324dee6e1e0f2d69  guix-build-ac9879c061af/output/x86_64-apple-darwin/bitcoin-ac9879c061af-x86_64-apple-darwin-unsigned.zip
649388ac163b907ee2f06df8b509e680cb48780731f4cf1691e613c92c317a18  guix-build-ac9879c061af/output/x86_64-apple-darwin/bitcoin-ac9879c061af-x86_64-apple-darwin.tar.gz
87ede93ae8624cdb2ce71218aa18d0f94e0374a826d52706293e712f1b64069b  guix-build-ac9879c061af/output/x86_64-linux-gnu/SHA256SUMS.part
fbb1f29aff36c07101bd7db3539e536ee33007f3f2791c4e2ba1fe3d6a0dd26c  guix-build-ac9879c061af/output/x86_64-linux-gnu/bitcoin-ac9879c061af-x86_64-linux-gnu-debug.tar.gz
e8e051e6c991e77206e75c1bbf9d7930cee23d522a57068142089bb2c2988857  guix-build-ac9879c061af/output/x86_64-linux-gnu/bitcoin-ac9879c061af-x86_64-linux-gnu.tar.gz
9bb68a94853b8b7c8629b16dcc5c1f9089e9e17249f22177e66d12e26623e013  guix-build-ac9879c061af/output/x86_64-w64-mingw32/SHA256SUMS.part
d746e34a5bae36d86f8ed05285594c3903010a11b0eb46254ef601ccea0da94c  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64-debug.zip
fccf90b415f1d7f6dc443c8fca389252f2e2cc6f4a61de511c146c1a96e29359  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64-setup-unsigned.exe
18f3ee8e0c7f515f8c8bd182a6e8ddd35bc4d116e0d5f16f133215cbb6312cf5  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64-unsigned.tar.gz
81126a896f72d910602796ebb2ca466e7ad1a16c2578028c941d6e5eb38a981f  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64.zip
```

**Comment by hebasto** [comment]
My Guix build:
```
x86_64
cdb3ff2cc9ee2ac4a60c8921359396c7b5b83447b54d223fa767c3e641bac182  guix-build-ac9879c061af/output/aarch64-linux-gnu/SHA256SUMS.part
58befd66969efb6473dc68ad9476334bf7544c4827ec28eae28437263b248020  guix-build-ac9879c061af/output/aarch64-linux-gnu/bitcoin-ac9879c061af-aarch64-linux-gnu-debug.tar.gz
53c6dd645f81be3bc3f52768331503c67034db17618fedd344e1283e50700caa  guix-build-ac9879c061af/output/aarch64-linux-gnu/bitcoin-ac9879c061af-aarch64-linux-gnu.tar.gz
7b9aa3aaa5ec51c7d8036abb94eb1f23195049022ee43ee11d401bc86f2990d9  guix-build-ac9879c061af/output/arm-linux-gnueabihf/SHA256SUMS.part
e67d97e27791d8323a4e958dba07d8b5ad220d20d525b77c223e1d13a139e555  guix-build-ac9879c061af/output/arm-linux-gnueabihf/bitcoin-ac9879c061af-arm-linux-gnueabihf-debug.tar.gz
bf621a9c9bf2f5a2a827e552766df7de6f0f4610f8d22a9b776b47e9f878af04  guix-build-ac9879c061af/output/arm-linux-gnueabihf/bitcoin-ac9879c061af-arm-linux-gnueabihf.tar.gz
fb2ecccd91e4154a47d7f509bd4f8255a3fd99e5bcc0af8a0d94ac7a16dd083f  guix-build-ac9879c061af/output/arm64-apple-darwin/SHA256SUMS.part
a3d592d77e231d8fded035a9f388f7c19285178c301d7c8e75e1510ad4ad399d  guix-build-ac9879c061af/output/arm64-apple-darwin/bitcoin-ac9879c061af-arm64-apple-darwin-unsigned.tar.gz
7d6bf27ee6568e4507847cb9d46dc25e35e79d9b1441807003d41b26a962ec5f  guix-build-ac9879c061af/output/arm64-apple-darwin/bitcoin-ac9879c061af-arm64-apple-darwin-unsigned.zip
84ed55538e7ff2baf9436fb277e9a92a2791e17a976b9ef8316daf36fc65096b  guix-build-ac9879c061af/output/arm64-apple-darwin/bitcoin-ac9879c061af-arm64-apple-darwin.tar.gz
9ba85fd31ebc9b7056345fb33b1ecdfe2c2b5be0fb34900be4f680145dcd705c  guix-build-ac9879c061af/output/dist-archive/bitcoin-ac9879c061af.tar.gz
c4a82c0e42e5f950d1c11df0e4fb17b417bd402261103ee409fad52d94042a00  guix-build-ac9879c061af/output/powerpc64-linux-gnu/SHA256SUMS.part
4bfa02a566b0862a027eca2d2b72746f3202dd99a20f912d05532e2d552a30bb  guix-build-ac9879c061af/output/powerpc64-linux-gnu/bitcoin-ac9879c061af-powerpc64-linux-gnu-debug.tar.gz
4594b1f0b8a64f795c7d1b88071e1844f7580cb8217973673212a533d218eb68  guix-build-ac9879c061af/output/powerpc64-linux-gnu/bitcoin-ac9879c061af-powerpc64-linux-gnu.tar.gz
790d95eb1a55d6bfe540abd713004118158480438a2a80d468a7fd781ff1f8a2  guix-build-ac9879c061af/output/riscv64-linux-gnu/SHA256SUMS.part
d54493b26d33cd144cf0376669efba365e6fc59cc2e4bf8bbdaf374373067460  guix-build-ac9879c061af/output/riscv64-linux-gnu/bitcoin-ac9879c061af-riscv64-linux-gnu-debug.tar.gz
8bd52003a4b01660916a97234ab172d3b8d4bfe90deea9025db54648c977f274  guix-build-ac9879c061af/output/riscv64-linux-gnu/bitcoin-ac9879c061af-riscv64-linux-gnu.tar.gz
38bc5ba2db9a2baf8b4be8cd80571a733ce11396a183313d2e5eda4390e25e12  guix-build-ac9879c061af/output/x86_64-apple-darwin/SHA256SUMS.part
6a71afeef1b80640cb830ab6c3b2fc4b3b5636dd20eb927aac39576e741f4dda  guix-build-ac9879c061af/output/x86_64-apple-darwin/bitcoin-ac9879c061af-x86_64-apple-darwin-unsigned.tar.gz
f9f9cddcfe2ca78998c77aa2feb580a5dfbd7e04b14a52a6324dee6e1e0f2d69  guix-build-ac9879c061af/output/x86_64-apple-darwin/bitcoin-ac9879c061af-x86_64-apple-darwin-unsigned.zip
649388ac163b907ee2f06df8b509e680cb48780731f4cf1691e613c92c317a18  guix-build-ac9879c061af/output/x86_64-apple-darwin/bitcoin-ac9879c061af-x86_64-apple-darwin.tar.gz
87ede93ae8624cdb2ce71218aa18d0f94e0374a826d52706293e712f1b64069b  guix-build-ac9879c061af/output/x86_64-linux-gnu/SHA256SUMS.part
fbb1f29aff36c07101bd7db3539e536ee33007f3f2791c4e2ba1fe3d6a0dd26c  guix-build-ac9879c061af/output/x86_64-linux-gnu/bitcoin-ac9879c061af-x86_64-linux-gnu-debug.tar.gz
e8e051e6c991e77206e75c1bbf9d7930cee23d522a57068142089bb2c2988857  guix-build-ac9879c061af/output/x86_64-linux-gnu/bitcoin-ac9879c061af-x86_64-linux-gnu.tar.gz
9bb68a94853b8b7c8629b16dcc5c1f9089e9e17249f22177e66d12e26623e013  guix-build-ac9879c061af/output/x86_64-w64-mingw32/SHA256SUMS.part
d746e34a5bae36d86f8ed05285594c3903010a11b0eb46254ef601ccea0da94c  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64-debug.zip
fccf90b415f1d7f6dc443c8fca389252f2e2cc6f4a61de511c146c1a96e29359  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64-setup-unsigned.exe
18f3ee8e0c7f515f8c8bd182a6e8ddd35bc4d116e0d5f16f133215cbb6312cf5  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64-unsigned.tar.gz
81126a896f72d910602796ebb2ca466e7ad1a16c2578028c941d6e5eb38a981f  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64.zip
```

**Comment by tobtoht** [comment]
```
$ uname --machine && find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
x86_64
cdb3ff2cc9ee2ac4a60c8921359396c7b5b83447b54d223fa767c3e641bac182  guix-build-ac9879c061af/output/aarch64-linux-gnu/SHA256SUMS.part
58befd66969efb6473dc68ad9476334bf7544c4827ec28eae28437263b248020  guix-build-ac9879c061af/output/aarch64-linux-gnu/bitcoin-ac9879c061af-aarch64-linux-gnu-debug.tar.gz
53c6dd645f81be3bc3f52768331503c67034db17618fedd344e1283e50700caa  guix-build-ac9879c061af/output/aarch64-linux-gnu/bitcoin-ac9879c061af-aarch64-linux-gnu.tar.gz
7b9aa3aaa5ec51c7d8036abb94eb1f23195049022ee43ee11d401bc86f2990d9  guix-build-ac9879c061af/output/arm-linux-gnueabihf/SHA256SUMS.part
e67d97e27791d8323a4e958dba07d8b5ad220d20d525b77c223e1d13a139e555  guix-build-ac9879c061af/output/arm-linux-gnueabihf/bitcoin-ac9879c061af-arm-linux-gnueabihf-debug.tar.gz
bf621a9c9bf2f5a2a827e552766df7de6f0f4610f8d22a9b776b47e9f878af04  guix-build-ac9879c061af/output/arm-linux-gnueabihf/bitcoin-ac9879c061af-arm-linux-gnueabihf.tar.gz
fb2ecccd91e4154a47d7f509bd4f8255a3fd99e5bcc0af8a0d94ac7a16dd083f  guix-build-ac9879c061af/output/arm64-apple-darwin/SHA256SUMS.part
a3d592d77e231d8fded035a9f388f7c19285178c301d7c8e75e1510ad4ad399d  guix-build-ac9879c061af/output/arm64-apple-darwin/bitcoin-ac9879c061af-arm64-apple-darwin-unsigned.tar.gz
7d6bf27ee6568e4507847cb9d46dc25e35e79d9b1441807003d41b26a962ec5f  guix-build-ac9879c061af/output/arm64-apple-darwin/bitcoin-ac9879c061af-arm64-apple-darwin-unsigned.zip
84ed55538e7ff2baf9436fb277e9a92a2791e17a976b9ef8316daf36fc65096b  guix-build-ac9879c061af/output/arm64-apple-darwin/bitcoin-ac9879c061af-arm64-apple-darwin.tar.gz
9ba85fd31ebc9b7056345fb33b1ecdfe2c2b5be0fb34900be4f680145dcd705c  guix-build-ac9879c061af/output/dist-archive/bitcoin-ac9879c061af.tar.gz
c4a82c0e42e5f950d1c11df0e4fb17b417bd402261103ee409fad52d94042a00  guix-build-ac9879c061af/output/powerpc64-linux-gnu/SHA256SUMS.part
4bfa02a566b0862a027eca2d2b72746f3202dd99a20f912d05532e2d552a30bb  guix-build-ac9879c061af/output/powerpc64-linux-gnu/bitcoin-ac9879c061af-powerpc64-linux-gnu-debug.tar.gz
4594b1f0b8a64f795c7d1b88071e1844f7580cb8217973673212a533d218eb68  guix-build-ac9879c061af/output/powerpc64-linux-gnu/bitcoin-ac9879c061af-powerpc64-linux-gnu.tar.gz
790d95eb1a55d6bfe540abd713004118158480438a2a80d468a7fd781ff1f8a2  guix-build-ac9879c061af/output/riscv64-linux-gnu/SHA256SUMS.part
d54493b26d33cd144cf0376669efba365e6fc59cc2e4bf8bbdaf374373067460  guix-build-ac9879c061af/output/riscv64-linux-gnu/bitcoin-ac9879c061af-riscv64-linux-gnu-debug.tar.gz
8bd52003a4b01660916a97234ab172d3b8d4bfe90deea9025db54648c977f274  guix-build-ac9879c061af/output/riscv64-linux-gnu/bitcoin-ac9879c061af-riscv64-linux-gnu.tar.gz
38bc5ba2db9a2baf8b4be8cd80571a733ce11396a183313d2e5eda4390e25e12  guix-build-ac9879c061af/output/x86_64-apple-darwin/SHA256SUMS.part
6a71afeef1b80640cb830ab6c3b2fc4b3b5636dd20eb927aac39576e741f4dda  guix-build-ac9879c061af/output/x86_64-apple-darwin/bitcoin-ac9879c061af-x86_64-apple-darwin-unsigned.tar.gz
f9f9cddcfe2ca78998c77aa2feb580a5dfbd7e04b14a52a6324dee6e1e0f2d69  guix-build-ac9879c061af/output/x86_64-apple-darwin/bitcoin-ac9879c061af-x86_64-apple-darwin-unsigned.zip
649388ac163b907ee2f06df8b509e680cb48780731f4cf1691e613c92c317a18  guix-build-ac9879c061af/output/x86_64-apple-darwin/bitcoin-ac9879c061af-x86_64-apple-darwin.tar.gz
87ede93ae8624cdb2ce71218aa18d0f94e0374a826d52706293e712f1b64069b  guix-build-ac9879c061af/output/x86_64-linux-gnu/SHA256SUMS.part
fbb1f29aff36c07101bd7db3539e536ee33007f3f2791c4e2ba1fe3d6a0dd26c  guix-build-ac9879c061af/output/x86_64-linux-gnu/bitcoin-ac9879c061af-x86_64-linux-gnu-debug.tar.gz
e8e051e6c991e77206e75c1bbf9d7930cee23d522a57068142089bb2c2988857  guix-build-ac9879c061af/output/x86_64-linux-gnu/bitcoin-ac9879c061af-x86_64-linux-gnu.tar.gz
9bb68a94853b8b7c8629b16dcc5c1f9089e9e17249f22177e66d12e26623e013  guix-build-ac9879c061af/output/x86_64-w64-mingw32/SHA256SUMS.part
d746e34a5bae36d86f8ed05285594c3903010a11b0eb46254ef601ccea0da94c  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64-debug.zip
fccf90b415f1d7f6dc443c8fca389252f2e2cc6f4a61de511c146c1a96e29359  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64-setup-unsigned.exe
18f3ee8e0c7f515f8c8bd182a6e8ddd35bc4d116e0d5f16f133215cbb6312cf5  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64-unsigned.tar.gz
81126a896f72d910602796ebb2ca466e7ad1a16c2578028c941d6e5eb38a981f  guix-build-ac9879c061af/output/x86_64-w64-mingw32/bitcoin-ac9879c061af-win64.zip
```

**Review by sedited — APPROVED** [review]
ACK ac9879c061af9d14bccd1eec31fcd826bed172df

Getting the same guix hashes on aarch64 and x86_64.

**Comment by hebasto** [comment]
Running Android CI task locally fails:
```
$ env -i HOME="$HOME" PATH="$PATH" USER="$USER" bash -c 'FILE_ENV="./ci/test/00_setup_env_android.sh" ./ci/test_run_all.sh'
...
rm -f ../../../lib/libqtpcre2_arm64-v8a.a
/ci_container_base/depends/SDKs/android/ndk/23.2.8568313/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ar cqs ../../../lib/libqtpcre2_arm64-v8a.a .obj/pcre2_auto_possess.o .obj/pcre2_chartables.o .obj/pcre2_compile.o .obj/pcre2_config.o .obj/pcre2_context.o .obj/pcre2_dfa_match.o .obj/pcre2_error.o .obj/pcre2_extuni.o .obj/pcre2_find_bracket.o .obj/pcre2_jit_compile.o .obj/pcre2_maketables.o .obj/pcre2_match.o .obj/pcre2_match_data.o .obj/pcre2_newline.o .obj/pcre2_ord2utf.o .obj/pcre2_pattern_info.o .obj/pcre2_script_run.o .obj/pcre2_serialize.o .obj/pcre2_string_utils.o .obj/pcre2_study.o .obj/pcre2_substitute.o .obj/pcre2_substring.o .obj/pcre2_tables.o .obj/pcre2_ucd.o .obj/pcre2_valid_utf.o .obj/pcre2_xclass.o
make[4]: Leaving directory '/ci_container_base/depends/work/build/aarch64-linux-android/qt/5.15.13-dca21c092e9/qtbase/src/3rdparty/pcre2'
rm -f ../../../lib/libQt5Bootstrap.a
ar cqs ../../../lib/libQt5Bootstrap.a .obj/qlatincodec.o .obj/qtextcodec.o .obj/qutfcodec.o .obj/qendian.o .obj/qglobal.o .obj/qlogging.o .obj/qmalloc.o .obj/qnumeric.o .obj/qoperatingsystemversion.o .obj/qrandom.o .obj/qabstractfileengine.o .obj/qbuffer.o .obj/qdebug.o .obj/qdir.o .obj/qdiriterator.o .obj/qfile.o .obj/qfileinfo.o .obj/qfilesystementry.o .obj/qfilesystemengine.o .obj/qfsfileengine.o .obj/qfsfileengine_iterator.o .obj/qiodevice.o .obj/qipaddress.o .obj/qfiledevice.o .obj/qresource.o .obj/qtemporarydir.o .obj/qtemporaryfile.o .obj/qsavefile.o .obj/qstandardpaths.o .obj/qloggingcategory.o .obj/qloggingregistry.o .obj/qurl.o .obj/qurlidna.o .obj/qurlquery.o .obj/qurlrecode.o .obj/qcoreapplication.o .obj/qcoreglobaldata.o .obj/qmetatype.o .obj/qvariant.o .obj/qsystemerror.o .obj/qsharedmemory.o .obj/qsystemsemaphore.o .obj/quuid.o .obj/qcborcommon.o .obj/qcborstreamwriter.o .obj/qcborvalue.o .obj/qdatastream.o .obj/qjsoncbor.o .obj/qjsondocument.o .obj/qjsonobject.o .obj/qjsonarray.o .obj/qjsonvalue.o .obj/qjsonparser.o .obj/qjsonwriter.o .obj/qtextstream.o .obj/qxmlutils.o .obj/qxmlstream.o .obj/qbytearray.o .obj/qbytearraylist.o .obj/qbytearraymatcher.o .obj/qlocale.o .obj/qlocale_tools.o .obj/qregexp.o .obj/qstring.o .obj/qstringbuilder.o .obj/qstring_compat.o .obj/qstringlist.o .obj/qstringview.o .obj/qvsnprintf.o .obj/qcalendar.o .obj/qdatetime.o .obj/qgregoriancalendar.o .obj/qromancalendar.o .obj/qarraydata.o .obj/qbitarray.o .obj/qcommandlineparser.o .obj/qcommandlineoption.o .obj/qcryptographichash.o .obj/qhash.o .obj/qlist.o .obj/qmap.o .obj/qringbuffer.o .obj/qpoint.o .obj/qrect.o .obj/qsize.o .obj/qline.o .obj/qversionnumber.o .obj/qdom.o .obj/qxml.o .obj/qcore_unix.o .obj/qsharedmemory_posix.o .obj/qsharedmemory_systemv.o .obj/qsharedmemory_unix.o .obj/qsystemsemaphore_posix.o .obj/qsystemsemaphore_systemv.o .obj/qsystemsemaphore_unix.o .obj/qfilesystemengine_unix.o .obj/qfilesystemiterator_unix.o .obj/qfsfileengine_unix.o .obj/qstandardpaths_unix.o .obj/adler32.o .obj/compress.o .obj/crc32.o .obj/deflate.o .obj/gzclose.o .obj/gzlib.o .obj/gzread.o .obj/gzwrite.o .obj/infback.o .obj/inffast.o .obj/inflate.o .obj/inftrees.o .obj/trees.o .obj/uncompr.o .obj/zutil.o
make[4]: Leaving directory '/ci_container_base/depends/work/build/aarch64-linux-android/qt/5.15.13-dca21c092e9/qtbase/src/tools/bootstrap'
make[3]: Leaving directory '/ci_container_base/depends/work/build/aarch64-linux-android/qt/5.15.13-dca21c092e9/qtbase/src'
make[2]: *** [Makefile:51: sub-src-make_first] Error 2
make[2]: Leaving directory '/ci_container_base/depends/work/build/aarch64-linux-android/qt/5.15.13-dca21c092e9/qtbase'
make[1]: *** [Makefile:49: sub-qtbase-make_first] Error 2
make[1]: Leaving directory '/ci_container_base/depends/work/build/aarch64-linux-android/qt/5.15.13-dca21c092e9'
make: *** [funcs.mk:297: /ci_container_base/depends/work/build/aarch64-linux-android/qt/5.15.13-dca21c092e9/./.stamp_built] Error 1
make: Leaving directory '/ci_container_base/depends'
```

**Comment by fanquake** [comment]
> Running Android CI task locally fails:

Isn't that already broken? https://github.com/bitcoin/bitcoin/issues/29360.

**Comment by fanquake** [comment]
> It seems different, as here, the qt package fails to build.

Ok, however I don't really consider that a blocker, given you can't compile `bitcoin-qt` in any case. (also because it's a platform we don't ship releases for, have no CI coverage for, and has been broken for nearly 2 months and nobody has fixed it).

**Comment by hebasto** [comment]
> you can't compile `bitcoin-qt` in any case.

Right :cry:

**Review by hebasto — APPROVED** [review]
ACK ac9879c061af9d14bccd1eec31fcd826bed172df.

Suggesting to amend another patch:
```diff
--- a/depends/patches/qt/memory_resource.patch
+++ b/depends/patches/qt/memory_resource.patch
@@ -17,7 +17,7 @@ and https://bugreports.qt.io/browse/QTBUG-114316
 
 --- a/qtbase/src/corelib/global/qcompilerdetection.h
 +++ b/qtbase/src/corelib/global/qcompilerdetection.h
-@@ -1050,16 +1050,22 @@
+@@ -1055,16 +1055,22 @@
  #   endif // !_HAS_CONSTEXPR
  #  endif // !__GLIBCXX__ && !_LIBCPP_VERSION
  # endif // Q_OS_QNX
```

**Comment by fanquake** [comment]
> Suggesting to amend another patch:

Done.

**Review by sedited — APPROVED** [review]
Re-ACK 430f319f73894499c56021b70f2827475c3f0433

**Review by hebasto — APPROVED** [review]
re-ACK 430f319f73894499c56021b70f2827475c3f0433.

**Pull request merged**